src/main/AndroidManifest.xml

```diff
@@ -4,12 +4,15 @@
           xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
This permission has to be requested on Android 7+

Reply:
fixed